sass/sass_wrapper.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * A Sass compiler wrapper that supports bazel persistent worker protocol.
+ * 
+ * Bazel can spawn a persistent worker process that handles multiple invocations. It
+ * will also be backwards compatible with being run as a normal single-invocation command.
```

Comment:
Describing this as "backwards compatible" could be misleading—as I understand, it's specifically *not* compatible with being invoked the way the old command was, by passing CLI args directly to the process. I'd say something more like "It can also be invoked with an argument file to run once and exit."

Reply:
Done.

Reply:
This is still the same wording as before...

Reply:
Sorry about that, done